### PR TITLE
Restructure Getting Started docs

### PR DESCRIPTION
**Pull request by jbyers19**
### Proposed changes
This PR restructures our Getting Started docs to make them easier to follow.

image

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/nginx-service-mesh/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that all unit tests pass after adding my changes
- [x] I have updated necessary documentation
- [x] I have rebased my branch onto main
- [x] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

File: docs/content/get-started/platform-setup/supported-platforms.md
Comment (sjberman) [inline docs/content/get-started/platform-setup/supported-platforms.md:7]: ?

File: docs/content/get-started/uninstall/uninstall.md
Comment (sjberman) [inline docs/content/get-started/uninstall/uninstall.md:64]: Same comment as before.

File: docs/content/get-started/platform-setup/supported-platforms.md
Comment (jbyers19) [reply docs/content/get-started/platform-setup/supported-platforms.md:7]: 🤷

File: docs/netlify.toml
Comment (jbyers19) [reply docs/netlify.toml]: Oh, didn't mean to commit that. With the new structure it's easy to find the uninstall and upgrade docs. Don't think we want to start down the path of adding a Netlify redirect every time we move a doc.

**Review by sjberman — APPROVED** [review]
LGTM pending review from docs team

**Review by ADubhlaoich — APPROVED** [review]
Looks good to me!

**Pull request merged**
